mlir/include/mlir/Dialect/OpenMP/OpenMPOpsInterfaces.td

```diff
@@ -142,6 +142,40 @@ def LoopWrapperInterface : OpInterface<"LoopWrapperInterface"> {
   ];
 }
 
+def ComposableLoopWrapperInterface : OpInterface<"ComposableLoopWrapperInterface"> {
```

Comment:
I think `LoopWrapper` shouldn't be part of this interface's name, because it would still be applied to `omp.parallel` after we remove it from the list of loop wrapper operations. I can suggest `ComposableOpInterface` or something similar.